src/Migrations/Version20181209021833.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace BalticRobo\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
```

Comment:
You should use our `AbstractMigration`.
```suggestion
use BalticRobo\Api\Migrations\AbstractMigration;
```